spiketools/measures/conversions.py

```diff
@@ -28,7 +28,7 @@ def create_spike_train(spikes, fs=1000, length=None):
     Convert 6 spike times into a corresponding binary spike train
 
     >>> spikes = [0.002, 0.250, 0.500, 0.750, 1.000, 1.250, 1.500]
-    >>> create_spike_train(spikes)
+    >>> convert_times_to_train(spikes)
     array([0, 0, 1, ..., 0, 0, 1])
```

Comment:
I really like the idea of distinguishing `spike times` from `spike train` in the naming here. Similarly, maybe also change the `convert_isis_to_spikes` function below to something like `convert_isis_to_spiketimes` to be extra clear.

Reply:
Good catch! I like this idea - I've updated `convert_isis_to_spikes` to `convert_isis_to_times`